scripts/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+PYTHONPATH=.
```

Comment:
Do we need this? (Or was it necessary in HTTPCore eg just so that unasync could work, or something?)

Reply:
Good catch 🤔 I don't think this'll actually do anything without being `export`ed too.